cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -326,6 +334,7 @@ public void initialize(MasterEnvironmentContext context) throws IOException, Ill
     String namespace = podInfo.getNamespace();
     String cdapNamespace = conf.getOrDefault(NAMESPACE_KEY, DEFAULT_NAMESPACE);
     additionalSparkConfs = getSparkConfigurations(conf);
+    programCpuMultiplier = conf.getOrDefault(PROGRAM_CPU_MULTIPLIER, DEFAULT_PROGRAM_CPU_MULTIPLIER);
```

Comment:
similar question here, shouldn't we be looking in the namespace configs for overrides to this?

Reply:
This one is at program level limits vs namespace level limits.